WarehouseManagement.Core/Contracts/IEntryService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using WarehouseManagement.Common.Statuses;
+using WarehouseManagement.Core.DTOs.Entry;
+
+namespace WarehouseManagement.Core.Contracts;
+
+public interface IEntryService
+{
+    Task<EntryDto> GetByIdAsync(int id);
+    Task<IEnumerable<EntryDto>> GetAllAsync(int? zoneId, ZoneEntryStatuses[]? statuses);
```

Comment:
lets split this into `GetAll` and `GetAllByZone` or `GetAllInZone` something like that, its currently doing two things when you look into the service - either get all or get all in a zone.

Reply:
You're right. I'll do it your way.